### PR TITLE
User story 8

### PR DESCRIPTION
**Pull request by martinhaiss**
This resolves [#8](https://github.com/martinhaiss/capstone-project/projects/1#card-71873840)

I've built in a delete button, so each card (aka activity) can now be deleted. I also exchanged some icons and updated the data.json.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,7 +41,18 @@ function App({ data }) {
     setActivities(newActivities);
     saveToLocal("localActivities", newActivities);
   }
+  //------------Delete Button -----------------
```
Comment (mbosselmann) [inline src/App.js:44]: I think you should delete this comment. :)

File: src/App.js [hunk 2]
```diff
@@ -41,7 +41,18 @@ function App({ data }) {
...
+  function handleDeleteButton(id) {
+    const filteredActivites = activities.filter(
+      (activity) => activity.id !== id
+    );
+
+    setActivities(filteredActivites);
+    saveToLocal("localActivities", filteredActivites);
+  }
+
+  //------------------------------------------------
```
Comment (mbosselmann) [inline src/App.js:55]: same here

File: src/components/Card.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components/macro";
 import { useState } from "react";
 import JoinButton from "./JoinButton";
+import { VscTrash } from "react-icons/vsc";
```
Comment (mbosselmann) [inline src/components/Card.js:4]: You could rename this icon for better understanding. Maybe `import {VscTrash as delete} from "react-icons/vsc"`?

**Review by mbosselmann — COMMENTED** [review]
Great job! 💯  I just have two really small comments. :-)

File: src/App.js (hunk 1 above)
Comment (martinhaiss) [reply src/App.js:44]: Thx, I totally forgot that 😂

File: src/App.js (hunk 2 above)
Comment (martinhaiss) [reply src/App.js:55]: And another line of premium code gone... 😉

File: src/components/Card.js (hunk 3 above)
Comment (martinhaiss) [reply src/components/Card.js:4]: Thx, that's a really good idea! I changed it to {VscTrash as DeleteIcon}. 🤩

**Pull request merged**